bgstally/overlay.py

```diff
@@ -125,8 +120,6 @@ def display_indicator(self, frame_name: str, ttl_override: int = None, fill_colo
             border_colour: str = border_colour_override if border_colour_override else fi['border_colour']
             self.edmcoverlay.send_shape(f"bgstally-frame-{frame_name}", "rect", border_colour, fill_colour, int(fi['x']), int(fi['y']), int(fi['w']), int(fi['h']), ttl=ttl)
 
-            self.problem_displaying = False
-
```

Comment:
I think this line needs to stay in.

If it's not reset to `False` when overlay text is displayed successfully, then in the situation where there is a problem displaying overlay text, followed by successful display, then another failure, the second failure will not be logged and may be missed when testing.